mkdud

```diff
@@ -2484,3 +2545,43 @@ sub sign_rpm
   # note: the 'setsid' is needed because rpmsign will try to read the passphrase from /dev/tty
   system "setsid rpmsign --addsign -D '%_gpg_path $sign_key_dir' -D '%_gpg_name $sign_key_id' $file </dev/null >$sign_key_dir/rpmsign.log 2>&1";
 }
+
+
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+# Create adddir script at given location.
+#
+# create_adddir(filename)
+#
+# The script lives here:
+#   https://github.com/openSUSE/installation-images/blob/master/data/initrd/adddir
```

Comment:
Um, I do not like much the idea about embedding a binary blob into the script.

I mean it's quite difficult to update it, you do not know which version is that, etc. So at least here should be the adddir version (date or commit SHA) so we know if that needs an update and how to update it.

If embedding the binary blob

Reply:
Ok, then I can live with that.